st2exporter/st2exporter/exporter/dumper.py

```diff
@@ -126,3 +135,33 @@ def _get_file_name(self):
         file_name = self._file_prefix + timestring + '.' + self._file_format
         file_name = os.path.join(self._export_dir, file_name)
         return file_name
+
+    def _update_marker(self, batch):
```

Comment:
We now have code for distributed locking in place so to be on the safe side we should also use it here when reading / writing the marker.